roles/brute-ora-cleanup/tasks/main.yml

```diff
@@ -220,6 +220,7 @@
     src: "{{ item.blk_device }}1"
     path: "{{ item.mount_point }}"
     state: absent
+  ignore_errors: true
```

Comment:
Wondering:
* if the mount removal can possibly fail for any other reasons (other than the covered directory here)?
* about the effect on the following blocks (that loop thru' the same `{{ oracle_user_data_mounts }}` variable) if the mount removal task `Remove Oracle user data devices from fstab` fails and we ignore it:
```
- name: Remove magic strings from Oracle user data disks
  become: yes
  become_user: root
  command: "wipefs --all --force {{ item.blk_device }}"
  with_items:
    - "{{ oracle_user_data_mounts }}"

- name: Reload partition table for user data disks
  become: yes
  become_user: root
  command: "partprobe {{ item.blk_device }}"
  with_items:
    - "{{ oracle_user_data_mounts }}"
```

Reply:
Changing to a more specific check for "Directory not empty" in the result, as requested.

/test bms-toolkit-install